source/Nevermore/Mapping/PrimitivePrimaryKeyHandler.cs

```diff
@@ -6,7 +6,7 @@ namespace Nevermore.Mapping
 {
     public abstract class PrimaryKeyHandler<T> : IPrimaryKeyHandler
     {
-        public Type Type => typeof(T);
+        public virtual Type Type => typeof(T);
```

Comment:
Does this need to be virtual? It is only meant to represent the underlying type, so I'm not sure it'd be a good idea for an inheriting class to override it.